app/src/main/java/com/example/movopfy/di/AppDi.kt

```diff
@@ -1,5 +1,11 @@
 package com.example.movopfy.di
 
+import com.example.movopfy.features.home.presentation.HomeViewModel
+import org.koin.androidx.viewmodel.dsl.viewModel
 import org.koin.dsl.module
 
-val appModule = module { }
\ No newline at end of file
+val appModule = module {
```

Comment:
In your home package create koin package, add val homeModule = module {...} and place all classes registration in that place that related to home (service, repository, viewmodel). Please remove dataDi, domainDi.

Reply:
did